extension/content_script.js

```diff
@@ -281,6 +282,15 @@ function highlight(sentenceId) {
 	highlightedSentenceId = sentenceId;
 };
 
+function persistentHighlight() {
+	let sentenceId = tracker.getSentenceId();
+	var el = doc.getSentenceEls(sentenceId)
+	if (el.attr('class').includes("persistentHighlight")) {
```

Comment:
Maybe use toggleClass?
https://api.jquery.com/toggleclass/

Reply:
Sorry missed this comment; will add